src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -109,34 +104,48 @@ public static TensorFlowTransform Create(IHostEnvironment env, ModelLoadContext
             Contracts.CheckValue(env, nameof(env));
             env.CheckValue(ctx, nameof(ctx));
             ctx.CheckAtModel(GetVersionInfo());
+
             // *** Binary format ***
+            // int: indicator for frozen models
             // stream: tensorFlow model.
             // int: number of input columns
             // for each input column
             //   int: id of int column name
             // int: number of output columns
             // for each output column
             //   int: id of output column name
-            byte[] modelBytes = null;
-            if (!ctx.TryLoadBinaryStream("TFModel", r => modelBytes = r.ReadByteArray()))
-                throw env.ExceptDecode();
-            var numInputs = ctx.Reader.ReadInt32();
-            env.CheckDecode(numInputs > 0);
-            string[] inputs = new string[numInputs];
-            for (int j = 0; j < inputs.Length; j++)
-                inputs[j] = ctx.LoadNonEmptyString();
+            var isFrozen = ctx.Reader.ReadInt32();
```

Comment:
>var isFrozen = ctx.Reader.ReadInt32(); [](start = 12, length = 38)

Need to increment the version number since the serialized format has changed. We can still read the older version by setting isFrozen=1 without reading anything.